patches/master_patch.patch

```diff
@@ -1782,6 +1782,19 @@ index e54e2c0fcc8eac6fe637025c3c71f3d6834a4892..d817176d073d8a96fa76b944f6fb198d
  };
  
  // Details about a redirection of a resource request.
+diff --git a/content/public/common/web_preferences.cc b/content/public/common/web_preferences.cc
+index 1e00ba53c4afcab6ccc747c255b032dccc373eb2..45693a865882770f19018eb030eb98d7a07ac382 100644
+--- a/content/public/common/web_preferences.cc
++++ b/content/public/common/web_preferences.cc
+@@ -227,7 +227,7 @@ WebPreferences::WebPreferences()
+       presentation_receiver(false),
+       media_controls_enabled(true),
+       do_not_update_selection_on_mutating_selection_range(false),
+-#if defined(OS_ANDROID)
++#if defined(OS_ANDROID) || defined(MUON_CHROMIUM_BUILD)
```

Comment:
MUON_CHROMIUM_BUILD is not defined for our builds, it is only defined when building Chromium. Is that correct here?